tests/shell/shell_recorder/listtest.esr

```diff
@@ -16,4 +16,5 @@ RET: 0
 
 < kdb ls $Mountpoint
 
+< kdb rm /sw/kdb/current/plugins
```

Comment:
I tried to backup and restore the data in commit f3b9900. However, it looks like `kdb cp` does not work correctly. The example below shows what I mean.

```sh
# 🐣 Create new test data
kdb set /sw/kdb/current/plugins 🦄
#> Using name user/sw/kdb/current/plugins
#> Create a new key user/sw/kdb/current/plugins with string 🦄
kdb get /sw/kdb/current/plugins
#> 🦄

# 💾 Copy the data using a cascading namespace
kdb cp /sw/kdb/current/plugins /backup/plugins

# 😢 Looks like this does not work
kdb get /backup/plugins
# RET: 1
# STDERR: Did not find key

# 😭 The command `kdb cp` did not create new data, although
# it exited successfully.
kdb ls user
#> user/sw/kdb/current/plugins

# 🤕 Let us try again — this time using an absolute namespace.
kdb cp /sw/kdb/current/plugins user/backup/plugins
# 🤔 Looks like this does not work either.
kdb get user/backup/plugins
# RET: 1
# STDERR: Did not find key

# 😱 Oh, there is our copy
kdb ls user
#> user/backup/pluginsgins
#> user/sw/kdb/current/plugins
kdb get user/backup/pluginsgins
#> 🦄
```